src/Nncase.Tests/Quant/UnitTestAddRangeOfMarker.cs

```diff
@@ -62,6 +62,54 @@ public void TestAddRangeOfMarkerToLeaky()
         Assert.Equal(1.00005388f, ((TensorConst)dumpVisitor.ExpressionMemo.Keys.ToList()[6]).Value.ToArray<float>()[1]);
     }
 
+    [Fact]
+    public void TestAddRangeOfMarkerToRelu6()
```

Comment:
The test code body is almost the same for each marker, the common parts need to be extracted